src/main/java/com/denizenscript/depenizen/bukkit/objects/breweryx/BreweryRecipeTag.java

```diff
@@ -0,0 +1,349 @@
+            return new ElementTag(object.bRecipe.getDifficulty());
+        });
+
+        // <--[tag]
+        // @attribute <BRecipeTag.cooking_time>
+        // @returns ElementTag(Number)
+        // @plugin Depenizen, BreweryX
+        // @description
+        // Returns the cooking time of the recipe.
+        // -->
+        tagProcessor.registerTag(ElementTag.class, "cooking_time", (attribute, object) -> {
+            return new ElementTag(object.bRecipe.getCookingTime());
+        });
+
+        // <--[tag]
+        // @attribute <BRecipeTag.distill_runs>
+        // @returns ElementTag(Number)
+        // @plugin Depenizen, BreweryX
+        // @description
+        // Returns the distill runs of the recipe
+        // -->
+        tagProcessor.registerTag(ElementTag.class, "distill_runs", (attribute, object) -> {
+            return new ElementTag(object.bRecipe.getDistillRuns());
+        });
+
+        // <--[tag]
+        // @attribute <BRecipeTag.distill_time>
+        // @returns ElementTag(Number)
+        // @plugin Depenizen, BreweryX
+        // @description
+        // Returns the amount of time each distill run takes.
+        // -->
+        tagProcessor.registerTag(ElementTag.class, "distill_time", (attribute, object) -> {
+            return new ElementTag(object.bRecipe.getDistillTime());
+        });
+
+        // <--[tag]
+        // @attribute <BRecipeTag.wood>
+        // @returns ElementTag(Number)
+        // @plugin Depenizen, BreweryX
+        // @description
+        // Returns the type of wood used in the recipe (by number, Ex: 0 = Any, 1 = Oak).
+        // -->
+        tagProcessor.registerTag(ElementTag.class, "wood", (attribute, object) -> {
+            return new ElementTag(object.bRecipe.getWood());
+        });
+
+        // <--[tag]
+        // @attribute <BRecipeTag.age>
+        // @returns ElementTag(Number)
+        // @plugin Depenizen, BreweryX
+        // @description
+        // Returns the amount of minecraft days the potion must age in a Brewery barrel.
+        // -->
+        tagProcessor.registerTag(ElementTag.class, "age", (attribute, object) -> {
+            return new ElementTag(object.bRecipe.getAge());
+        });
+
+        // <--[tag]
+        // @attribute <BRecipeTag.color>
+        // @returns ColorTag
+        // @plugin Depenizen, BreweryX
+        // @description
+        // Returns the color of the distilled/finished potion.
+        // -->
+        tagProcessor.registerTag(ColorTag.class, "color", ((attribute, object) -> {
+            Color color = object.bRecipe.getColor().getColor();
+            return new ColorTag(color.getRed(), color.getGreen(), color.getBlue(), color.getAlpha());
+        }));
+
+        // <--[tag]
+        // @attribute <BRecipeTag.alcohol>
+        // @returns ElementTag(Number)
+        // @plugin Depenizen, BreweryX
+        // @description
+        // Returns the amount of alcohol in a perfect potion.
```

Comment:
What unit is this? Item stacks of an alcohol item, percentage, decimal, ...?

Reply:
"alcohol: Absolute amount of alcohol 0-100 in a perfect potion (will be added directly to the player, where 100 means fainting)"

Just returns the int written in Brewery's config for the specific recipe. I'll add that longer description to the comment